x-pack/plugins/observability_solution/synthetics/e2e/synthetics/journeys/overview_sorting.journey.ts

```diff
@@ -13,12 +13,7 @@ import {
 } from './services/add_monitor';
 import { syntheticsAppPageProvider } from '../page_objects/synthetics_app';
 
-const journeySkip =
-  (...params: Parameters<typeof journey>) =>
-  () =>
-    journey(...params);
-// TODO: skipped because failing on main and need to unblock CI
-journeySkip('OverviewSorting', async ({ page, params }) => {
+journey('OverviewSorting', async ({ page, params }) => {
```

Comment:
Hey Justin, what issue did you solve? I'm having trouble understanding the main change.

Was this a flaky test, and does it make sense to run a flaky test runner for this PR?

Reply:
I am unsure why this test was originally skipped, but the reason it is failing now is we made some changes to the page it tests and the test ID was not rendered anymore.

I [re-introduced that test ID](https://github.com/elastic/kibana/pull/192614/files#diff-2ebd78c32d4e6a9889c66c45c5114e74895d86e965023372154dcd6d3d944780R203) and didn't need to modify the test to get it working again.

We can run it through the flaky test runner, sure. I haven't done this in a while, I'm assuming it's not on Jenkins anymore.

Reply:
I looked at the flaky build success above and it appears that the build was flaky because of a Jest failure and an unrelated e2e test failure in the SLO plugin; I think it's unlikely this test is flaky but we can give it a try if you like.

Reply:
My comment was not related to the PR flaky build but to the current test and ensuring it is not flaky.
If the test was failing before and was not flaky, up to you if running the flaky test runner makes sense :)